bin/appliance_console

```diff
@@ -383,7 +383,8 @@ Static Network Configuration
           "Create Internal Database"           => "create_internal",
           "Create Region in External Database" => "create_external",
           "Join Region in External Database"   => "join_external",
-          "Reset Configured Database"          => "reset_region"
+          "Reset Configured Database"          => "reset_region",
+          "No changes"                         => "no_changes"
```

Comment:
Why add this? Is this just in case someone wants to configure kafka but not the database? Would it be less confusing as a new top-level item or do we want to keep all the "initial setup"-type stuff in one wizard-style menu?

Reply:
Also, if we're keeping it this way, typically we ask the user for all of the information then offer a summary. Only then do we actually go and do any of the work.

Do we want to ask for the database info, then ask for the kafka info, then do whatever we need to?

Reply:
I think I discussed the flow with @Fryguy a while ago and we thought that the database and queue configuration should be tightly coupled and that this should be "application configuration" as opposed to the other things in the menu which are OS configuration.  So, I thought it would be a good idea to be able to skip database changes if you only wanted to make queue changes.

Reply:
> Do we want to ask for the database info, then ask for the kafka info, then do whatever we need to?

Probably a good idea.  I'll change it